keyboards/40percentclub/gherkin/kb2040/readme.md

```diff
@@ -0,0 +1,22 @@
+# Gherkin
+
+![Gherkin](https://4.bp.blogspot.com/-sQ18-lNZXOc/WCzlTde-4PI/AAAAAAAB_JQ/qQrehAMG6DMKf3i4oj4mkmLGOfTUvb3KgCLcB/s640/IMG_20161116_122926.jpg)
```

Comment:
Please rehost on imgur.

Reply:
I used the same URL already present in the parent readme:
```
keyboards/40percentclub/gherkin/kb2040/readme.md:![Gherkin](https://4.bp.blogspot.com/-sQ18-lNZXOc/WCzlTde-4PI/AAAAAAAB_JQ/qQrehAMG6DMKf3i4oj4mkmLGOfTUvb3KgCLcB/s640/IMG_20161116_122926.jpg)
keyboards/40percentclub/gherkin/readme.md:![Gherkin](https://4.bp.blogspot.com/-sQ18-lNZXOc/WCzlTde-4PI/AAAAAAAB_JQ/qQrehAMG6DMKf3i4oj4mkmLGOfTUvb3KgCLcB/s640/IMG_20161116_122926.jpg)
```

Reply:
The original readme is from ~4 years ago, before we added a bunch of the policies.  

Eg, that readme should be fixed too, but would consider that to be outside the scope of this PR.

Reply:
I have re-hosted both images on imgur